modules/default/clock/clock.js

```diff
@@ -199,13 +199,13 @@ Module.register("clock", {
 			sunWrapper.innerHTML =
 				'<span class="' +
 				(isVisible ? "bright" : "") +
-				'"><i class="fa fa-sun-o" aria-hidden="true"></i> ' +
+				'"><i class="fas fa-sun-o" aria-hidden="true"></i> ' +
```

Comment:
```suggestion
				'"><i class="fas fa-sun" aria-hidden="true"></i> ' +
```

<img width="124" alt="Bildschirmfoto 2022-01-05 um 13 37 09" src="https://user-images.githubusercontent.com/9592452/148219410-73a1ed7c-de4e-41fb-8c33-bb228adcfda9.png">
<img width="159" alt="Bildschirmfoto 2022-01-05 um 13 37 03" src="https://user-images.githubusercontent.com/9592452/148219404-66df9580-647c-43e1-8341-49ae36143853.png">